### PR TITLE
Move orgs and users to the "Your account" page

### PR DESCRIPTION
**Pull request by benswannack**
## Context
With notifications coming up soon, we are moving account-related provider content to live under the account section. This includes the current profile page, the users page and the organisations page. We have also renamed organisations to organisational permissions.

## Changes proposed in this pull request
The base of this PR is to move the nav links for Orgs and Users to be in the account page. Along the way, there were a few things I cleaned up, including: The org permissions setup success screen, adding breadcrumbs and page titles in the users and orgs pages

### Page header now looks like:

image

### Account page:

image

### If the user does not have manage users or orgs:

image

### Users page (and breadcrumbs):

image

### Success page for permission setup:

image

## Guidance to review
I copied the routing, breadcrumbs and page titles from the prototype. Have I missed anything?

## Link to Trello card
https://trello.com/c/GNqpIZAr/2980-put-users-and-organisational-permissions-into-the-account-area

## Things to check

- [x] This code does not rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/views/provider_interface/provider_relationship_permissions/edit.html.erb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <% content_for :browser_title, title_with_error_prefix('', @form.errors.any?) %>
+<% content_for :before_content, govuk_back_link_to(provider_interface_organisation_path(@form.training_provider)) %>
```
Comment (benswannack) [inline app/views/provider_interface/provider_relationship_permissions/edit.html.erb:2]: Hopefully this is fine, as I don't think you could go from the ratifying provider page to the edit permissions page?

Comment (benswannack) [reply app/views/provider_interface/provider_relationship_permissions/edit.html.erb:2]: Oh, I suppose I can also just not specify where that back link goes

**Comment by adamsilver** [comment]
Until we have notifications can you use one column for all items please.

File: app/views/provider_interface/profile/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<%= content_for :browser_title, 'Profile' %>
```
Comment (duncanjbrown) [inline app/views/provider_interface/profile/show.html.erb:1]: Might as well `i18n` this while we're here

File: app/views/provider_interface/profile/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+<% content_for :before_content do %>
+  <%= render BreadcrumbComponent.new(items: [
+    {
+      text: t('page_titles.provider.account'),
+      path: provider_interface_account_path
+    },
+    {
+      text: 'Profile'
```
Comment (duncanjbrown) [inline app/views/provider_interface/profile/show.html.erb:9]: And here

**Review by duncanjbrown — APPROVED** [review]

**Pull request merged**